SUI/SUI/ContentView.swift

```diff
@@ -5,14 +5,98 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let moneyCount = 465
+        static let walletString = "Кошелек"
+        static let currentBalanceText = "Текущий баланс:"
+        static let showSettingsText = "Показать настройки"
+        static let emptyString = ""
+        static let refillTheBalanceButtonTitle = "Пополнить баланс"
+        static let actionSheetRefillTheBalanceText = "Вы пополнили счёт на 1000"
+        static let transferMoneyTitle = "Перевод средств"
+        static let transferMoneyMessage = "Перевeсти 500 деняк?"
+        static let profileTitle = "Профиль"
+        static let thousandNumber = 1000
+        static let fiveHundredNumber = 500
+        static let spacerEightyNumber = 80
+        static let spacerFourNumber = 400
+        static let toggleNumber = 150
+        static let zeroNumber = 0
+        static let cornerRadiusNumber = 10
+        static let transfersMoneyAlertButton = "Перевести"
+        static let openBankAccountText = "Открыть счёт"
+        static let responseNumber = 0.5
+        static let dampingFractionNumber = 0.7
+        static let blendDurationNumber = 0.3
+    }
+
+    @State var isError = false
+    @State var isOnToggle = false
+    @State var isAction = false
```

Comment:
```suggestion
    @State var isActionShown = false
```
везде поправь Shown где переход